ontology/yaml/resources/FIRE/entity_types/ABSTRACT.yaml

```diff
@@ -0,0 +1,1177 @@
+FSDSM:
```

Comment:
If these logical devices only have a status feedback, they should all be modeled that way and have virtual types defined.